docs/server/requirements.txt

```diff
@@ -3,3 +3,5 @@ recommonmark>=0.4.0
 sphinx-rtd-theme>=0.1.9
 sphinxcontrib-napoleon>=0.4.4
 sphinxcontrib-httpdomain>=1.5.0
+pyyaml
```

Comment:
any version of `pyyaml` will do?

Reply:
Oh, good question. I'll put pyyaml>=3.12